packages/subproviders/CHANGELOG.json

```diff
@@ -1,4 +1,13 @@
 [
+    {
+        "timestamp": 1601647297,
```

Comment:
```suggestion
```

This is automatic on a publish, so need to remove otherwise our infra thinks it has been published :)

Reply:
Haha yes this felt strange to me, but I was trying to work through the contribution guidelines.

Reverted in b58b3a6.

(By the way, I can squash these commits if/when the PR is finalized/approved.)